src/main/java/io/luna/game/model/Area.java

```diff
@@ -195,7 +195,7 @@ public Position random() {
      * @return The center position.
      */
     public Position center() {
-        int halfWidth = width() / 2;
+        int halfWidth = width() >> 1;
```

Comment:
Divide operator is fine here since its more readable and this isn't a performance critical piece of code.

Reply:
Agreed; readability over performance.